src/app/pages/storage/stores/pools-dashboard-store.service.ts

```diff
@@ -50,6 +50,7 @@ export class PoolsDashboardStore extends ComponentStore<PoolsDashboardState> {
     private sorter: StorageService,
   ) {
     super(initialState);
+    this.subscribeToPoolsUpdate();
```

Comment:
I think it may be better to call this explicitly from outside (or call something like `poolDashboardEntered` that would call `loadDashboard` and this.